test/fixture/postgres/initCleanup.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS users;
+DROP TYPE IF EXISTS user_status_enum;
+DROP TYPE IF EXISTS format_enum;
+
+DROP SCHEMA IF EXISTS maxi CASCADE;
+DROP TABLE IF EXISTS maxi.users1;
```

Comment:
We can leave these individual drop table statements out since we dropped the maxi schema in cascading fashion in the query above.